### ISSUE TITLE
Usage information is not displayed for commands which contain nullable parameters.

### ISSUE THREAD

**Issue by andykernahan** [andykernahan/ak-cmdline#6]

**Comment by andykernahan** [comment]
`XmlDocUsageProvider` needs to support resolving the member for a method which contains generic parameters.

E.g.

```csharp
public class Program
{
    /// <summary>
    /// Method summary.
    /// </summary>
    /// <param name="a">Parameter summary.</param>
    public void Command(int? a = null) { }
}
```
Generates the following member:
```xml
<member name="M:Issue.Program.Command(System.Nullable{System.Int32})">
    <summary>
    Method summary.
    </summary>
    <param name="a">Parameter summary.</param>
</member>
```

**Comment by andykernahan** [comment]
Closed by #7.
